packages/puppeteer-core/src/api/Locator.ts

```diff
@@ -732,6 +732,26 @@ class RaceLocatorImpl extends Locator {
     );
 
     options.signal?.throwIfAborted();
+
+    if (
+      results.some(result => {
+        return result.status === 'fulfilled';
+      })
+    ) {
+      return;
+    }
+
+    for (const result of results) {
+      if (result.status === 'fulfilled') {
```

Comment:
How can we get in this block at all?

Reply:
currently no, but TypeScript does not know

Reply:
updated